src/cargo/sources/registry/mod.rs

```diff
@@ -235,14 +234,14 @@ enum Field {
 
 #[derive(Deserialize)]
 struct RegistryDependency<'a> {
-    name: Cow<'a, str>,
```

Comment:
I think for these the `Cow` was due to the derive above to handle escapes in names if they ever creep in

Reply:
So do you want me to leave the `cows` in, incase they are useful; or, do you want me to take them out and we can re-add them if needed?

Currently we are scrubbing data by reing the raw version from `RegistryDependency` then cleaning it into a local and using the local instead.